comprehensive-demo/e2e/checkComrehensiveDemoApp2.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { dialogs } from './../../cypress/common/selectors';
+import { BaseMethods } from "../../cypress/common/base";
+import { block, baseSelectors } from "../../cypress/common/selectors";
+import { Constants } from "../../cypress/fixtures/constants";
+
+const basePage: BaseMethods = new BaseMethods()
+
+describe('Check Comprehensive Demo App2', () => {
```

Comment:
add some more description. There are a lot of checks for such vague description

Reply:
Fixed